distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -454,6 +454,35 @@ type Column
     not : Column
     not self = self.make_unary_op "NOT"
 
+    ## UNSTABLE
+       Replaces `True` values with `when_true` and `False` with `when_false`.
+       Only meant for use with boolean columns.
+
+       TODO: Currently `when_true` and `when_false` need to be a single value.
+       In the future the API will also support row-based IIF if they are columns.
+    iif : Any -> Any -> Column
+    iif self when_true when_false =
```

Comment:
I decided to just implement a prototype, because full `iif` relies on Value Type works that is not completed yet. It correctly implements a subset of the true `iif` that is enough for our current internal use-case. I then plan to revisit it when we will be actually implementing the column operations, having already the type logic in place.